### ISSUE TITLE
Add pagination to /address page

### ISSUE THREAD

**Issue by cryptoni9n** [ordinals/ord#3846]
I see a couple of issues in v0.19.0 on the address page:

**FIXED** 1) from #3831 - some runes are not being displayed/linked correctly.  [Example 1](http://ordinals.com/address/bc1pvjx6xqa4rkvxc5suw6q9zrj4hz7aqz6r3a06kp5dpqcht6jq8jnqekq3kr),  [Example 2](http://ordinals.com/address/bc1pjjzkh79z7v3c0phxcykcfuz774n84ef77gqpmv36a2vmsmd0nzrqqa46k6)

2) from #3843 - some addresses have a very large number of inscriptions and outputs.  these pages take forever to load and could be detrimental to the site.  I think pagination is needed: http://ordinals.com/address/bc1pafhpvjgj5x7era4cv55zdhpl57qvj0c60z084zsl7cwlmn3gq9tq3hqdmn

edit: reopening to keep track of pagination issue

**Comment by raphjaph** [comment]
Just updated ordinals.com. Yes, we definitely need pagination. Just wanted to get the release out so we skipped that. You could try your hand at that if you want :)

**Comment by cryptoni9n** [comment]
adding some more info here for the first issue - the rune display issue only seems to occur with runes that have the `¤` as their currency symbol.

**Comment by cryptoni9n** [comment]
reopening to keep track of pagination issue

**Comment by raphjaph** [comment]
Jup jup, we should add this

**Comment by alpeshvas** [comment]
How would pagination work though considering there are different type of assets in one page: UTXOs, inscriptions, runes etc…! Ideally all of these have their own unique identifier (URI) like 

/address/{addy}/inscriptions, /address/{addy}/inscriptions etc…

**Comment by twosatsmaxi** [comment]
> Jup jup, we should add this

How would pagination work though considering there are different type of assets in one page: UTXOs, inscriptions, runes etc…! Ideally all of these have their own unique identifier (URI) like

/address/{addy}/inscriptions, /address/{addy}/inscriptions etc…

**Comment by raphjaph** [comment]
> How would pagination work though considering there are different type of assets in one page: UTXOs, inscriptions, runes etc…! Ideally all of these have their own unique identifier (URI) like
> 
> 
> 
> /address/{addy}/inscriptions, /address/{addy}/inscriptions etc…

Yes, that's right. The /address/<ADDY> would show the latest 4 inscriptions, the first 10 runes (ordered lexicographically) and maybe the latest 10 outputs. There would be a 'more' button that then links to the pages mentioned by you. Each page having 100 entries or so. Happy to answer any more questions if you'd like to take a stab at it.
